guides/managing-current-user.md

```diff
@@ -23,24 +61,19 @@ const { inject: { service }, isEmpty, RSVP } = Ember;
 export default Ember.Service.extend({
   store: service(),
 
-  account: null,
-
   loadCurrentUser() {
-    return new RSVP.Promise((resolve, reject) => {
-      return this.get('store').find('user', 'me').then(
-        user => resolve(this.set('user', user)),
-        reject
-      );
+    return this.get('store').find('user', 'me').then((account) => {
```

Comment:
maybe we should keep the user variable name here to keep it consistent with the find action to keep things clear.